### DISCUSSION TITLE
Ratelimits to prevent mass nukes

### DISCUSSION THREAD

**Discussion by cheesycod** [piccolo-orm/piccolo_admin#453] [Ideas]
It would be nice if piccolo admin allowed ratelimiting non super users to protect against account compromises.

**Comment by dantownsend** [comment]
There's currently an option in `create_admin`, which is `rate_limit_provider`. This is just applied to the login endpoint. It's not documented very well at the moment.

Were you thinking of just protecting the login endpoint, or all endpoints?

**Comment by cheesycod** [comment]
All endpoints
